pkg/sfu/downtrack.go

```diff
@@ -161,15 +160,13 @@ func NewDownTrack(c webrtc.RTPCodecCapability, r TrackReceiver, bf *buffer.Facto
 	d.rtxStats.Store(new(PacketStats))
 	d.paddingStats.Store(new(PacketStats))
 
-	return d, nil
-}
-
-func (d *DownTrack) SetTrackType(isSimulcast bool) {
-	if isSimulcast {
+	if r.IsSimulcast() {
```

Comment:
@cnderrauber This will need a change in relay too.

Reply:
we can pass signal *livekit.trackInfo to Relay modules, it can aware of more information like simulcast, layers...